lib/App/index.js

```diff
@@ -679,12 +679,19 @@ class App {
    * @private
    */
   async _getPrunePaths() {
+    if (await NpmCommands.isNpmInstalled() === false) {
+      return;
+    }
+
     // Check if npm is available then start prune dry-run
-    const npmInstalled = await NpmCommands.isNpmInstalled();
-    if (npmInstalled) {
-      Log(colors.green('✓ Pruning dev dependencies...'));
-      return NpmCommands.getPrunePaths({path: this.path});
+    const npmVersion = await NpmCommands.getNpmVersion();
+
+    if (semver.lt('7.0.0', npmVersion)) {
```

Comment:
```suggestion
    if (semver.lte('7.0.0', npmVersion)) {
```

Not sure if there is a v7.0.0 floating around but lets just cover that as well